src/DDoSX/DomainClient.php

```diff
@@ -67,6 +68,32 @@ public function create(Domain $domain)
             });
     }
 
+    /**
+     * @param Domain $domain
+     * @return null
+     */
+    public function deploy(Domain $domain)
+    {
+        $this->post('v1/domains/' . $domain->name . '/deploy');
+
+        return null;
```

Comment:
We should return at least something here to indicate success, maybe `true`?

Reply:
updated to return true